### PR TITLE
add emtpy object to health api call

### PR DESCRIPTION
**Pull request by titivermeesch**
Agones updated the requirements for the health check. It now requires an empty object to correctly validate.

### CODE REVIEW

**Comment by titivermeesch** [comment]
Should something be done about the version update?

File: src/main/java/dev/agones/AgonesSDK.java [hunk 1]
```diff
@@ -64,7 +61,7 @@ public void startHealthTask(int period, TimeUnit unit) {
     }
 
     public void health() {
-        getEndpoints(AgonesEndpoints.class).health();
+        getEndpoints(AgonesEndpoints.class).health(new Empty());
```
Comment (theminecoder) [inline src/main/java/dev/agones/AgonesSDK.java]: I'm thinking this would be better as a singular static instance

**Review by theminecoder — CHANGES_REQUESTED** [review]

File: src/main/java/dev/agones/AgonesSDK.java (hunk 1 above)
Comment (MylesDev) [reply src/main/java/dev/agones/AgonesSDK.java]: Agreed. Potentially `Empty#empty` (or something similar?) returning the static instance.

Comment (titivermeesch) [reply src/main/java/dev/agones/AgonesSDK.java]: @theminecoder Not sure what you mean here. Do you mean something like @MylesDev pointed out?

Comment (theminecoder) [reply src/main/java/dev/agones/AgonesSDK.java]: Yeh something like that should suffice. The key point is we don't need to make a new `Empty` instance for each request

Comment (titivermeesch) [reply src/main/java/dev/agones/AgonesSDK.java]: Not sure how to structure this. Does this look right? Shouldn't it be initialized at least once somewhere?
```java
public class Empty {
    public static Empty empty;

    public Empty() {}
}
```

Comment (MylesDev) [reply src/main/java/dev/agones/AgonesSDK.java]:
```java
public class Empty {
    
    private static final Empty INSTANCE = new Empty();
    
    public static Empty empty() {
        return INSTANCE;
    }

    private Empty() {}
}
```

Comment (titivermeesch) [reply src/main/java/dev/agones/AgonesSDK.java]: > ```java
> public class Empty {
>     
>     private static final Empty INSTANCE = new Empty();
>     
>     public static Empty empty() {
>         return INSTANCE;
>     }
> 
>     private Empty() {}
> }
> ```

Thanks for that, seems very logical indeed. I had something like that already but was worried about my IDE complaining about a utility class instantiation.

I pushed the changes

**Comment by MylesDev** [comment]
Looks good to me!

**Review by theminecoder — APPROVED** [review]

**Pull request merged**